src/nghttp.cc

```diff
@@ -687,7 +687,9 @@ void HttpClient::disconnect() {
   state = ClientState::IDLE;
 
   for (auto req = std::begin(reqvec); req != std::end(reqvec); ++req) {
-    (*req)->continue_timer->stop();
+    if ((*req)->continue_timer) {
```

Comment:
Integration test for nghttp is somewhat long term goal.  No rush at the moment.  I still don't have clear picture how it will look like.